tests/integration/test_end_to_end_dcp.py

```diff
@@ -104,9 +104,8 @@ class TestSmartSeq2Run(TestEndToEndDCP):
     ]
 
     def test_smartseq2_run(self):
-        runner = self.ingest_store_and_analyze_dataset(dataset_fixture='Smart-seq2')
-
-        with Timeout(110 * 60) as to:  # timeout after 1 hour and 50 minutes
+        with Timeout(6) as to:  # timeout after 1 hour and 50 minutes
```

Comment:
@xbrianh I'm guessing you want to change this back to 110 * 60?

Reply:
fixed, thanks